.github/workflows/ci.yml

```diff
@@ -0,0 +1,103 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v*
+  pull_request:
+
+jobs:
+  test:
+    name: ${{ matrix.os.name }} ${{ matrix.python.name }} ${{ matrix.reactor.name }}
+    runs-on: ${{ matrix.os.runs-on }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - name: Linux
+            runs-on: ubuntu-latest
+          - name: Windows
+            runs-on: windows-latest
+          - name: macOS
+            runs-on: macos-latest
+        python:
+          - name: CPython 2.7
+            tox: py27
+            action: 2.7
+          - name: CPython 3.5
+            tox: py35
+            action: 3.5
+          - name: CPython 3.6
+            tox: py36
+            action: 3.6
+          - name: CPython 3.7
+            tox: py37
+            action: 3.7
+        reactor:
+          - name: default
+            tox: default
+          - name: Qt5
+            tox: qt5
+          - name: asyncio
+            tox: asyncio
+        exclude:
+          - python:
+              tox: py27
+            reactor:
+              tox: qt5
+          - python:
+              tox: py27
+            reactor:
+              tox: asyncio
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up ${{ matrix.python.name }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python.action }}
+          architecture: x64
+      - name: Install Linux dependencies
+        if: matrix.system.platform == 'linux'
+        run: |
+          sudo apt-get update
+          sudo apt-get install xvfb
```

Comment:
I suspect some config somewhere makes apt-get assume yes implicitly, but maybe make it explicit: `sudo apt-get install -y xvfb`

Reply:
Yes, thanks.

Reply:
Yes.